tests/server_test.js

```diff
@@ -9,10 +9,37 @@ test('Testing empty root get request - / ', t => {
     .expect('Content-Type', /json/)
     .expect(200)
     .end((err, res) => {
-        console.log('in end');
-        console.log(res.text)
+      console.log(res.text);
```

Comment:
We should avoid using console.logs

if you'll look at #17 i remove all the console.logs at the other files
we should find other way to get errors